src/components/WindowSideBarCanvasPanel.js

```diff
@@ -115,6 +115,7 @@ export class WindowSideBarCanvasPanel extends Component {
                     name="sequenceId"
                     classes={{ select: classes.select }}
                     className={classes.selectEmpty}
+                    data-testid="sequence-select"
```

Comment:
@phil-plencner-hl this is probably how you will need to access that inner-rendered component we were looking at -- add the data-testid directly in the component.